### ISSUE TITLE
Move example configs to opencontainers/specs?

### ISSUE THREAD

**Issue by wking** [opencontainers/runc#297]
I'd [suggested this][1] to keep things DRYer while looking over #242.  @LK4D4 [thought it deserved its own PR][2].  Now that #242 has landed, do folks who are maintaining both projects feel like it's a good time to make that transition?

[1]: https://github.com/opencontainers/runc/pull/242/files#r38484100
[2]: https://github.com/opencontainers/runc/pull/242/files#r38484776

**Comment by crosbymichael** [comment]
This is not the place to ask questions like this.  This can be done on IRC or we can just do the work.  You should not open issues like this.

**Comment by wking** [comment]
On Wed, Sep 23, 2015 at 04:34:01PM -0700, Michael Crosby wrote:
> This can be done on IRC or we can just do the work.

Resolved (on IRC) to punt for now:

16:39 < crosbymichael> i would stay the spec is still moving to much
  to have static examples commit into the repo
16:39 < crosbymichael> we can wait and have time to wait on doing this
16:40 < wking> there are already examples for most things in the spec.
  With a bit of tagging we may be able to automatically generate the
  full example.  Or just update the example at spec-tag-time?
16:42 < crosbymichael> i'm not sure what to do but it's nothing that
  we need now, i think it can wait so we are not trying to hit a
  moving target
16:42 < wking> ok
